### PR TITLE
Fix long query segments

### PR DESCRIPTION
**Pull request by Ostico**

### CODE REVIEW

File: public/js/cat_source/es6/actions/CatToolActions.js [hunk 1]
```diff
@@ -274,7 +274,7 @@ let CatToolActions = {
           entries: domains,
         })
       }
-      if (haveKeysGlossary) {
+      if (haveKeysGlossary !== undefined) {
```
Comment (Ostico) [inline public/js/cat_source/es6/actions/CatToolActions.js:277]: @piedicianni @riccio82 please check if this is correct

Comment (piedicianni) [reply public/js/cat_source/es6/actions/CatToolActions.js:277]: yes Dom, it's correct!

**Review by mauretto78 — DISMISSED** [review]

File: public/js/cat_source/es6/actions/CatToolActions.js (hunk 1 above)
Comment (Ostico) [reply public/js/cat_source/es6/actions/CatToolActions.js:277]: :+1:

File: lib/Model/Segments/SegmentDao.php [hunk 2]
```diff
@@ -1113,7 +1118,16 @@ public static function getSegmentsForAnalysisFromIdJobAndPassword($idJob, $passw
...
             LEFT JOIN
-                files_parts fp ON fp.id = s.id_file_part        
+                files_parts fp ON fp.id = s.id_file_part     
+            LEFT JOIN (
+                SELECT id_job, id_segment, group_concat( distinct source_page )  as _page
+                FROM segment_translation_events stex
+                JOIN
+                    jobs j ON stex.id_job = j.id
+                WHERE stex.id_job = j.id
+                    AND id_project = :id_project
```
Comment (Ostico) [inline lib/Model/Segments/SegmentDao.php:1128]: @mauretto78 this will fail, there is not id_project in thid method signature

**Review by piedicianni — APPROVED** [review]

**Pull request merged**